### PR TITLE
Add initial changelog for 1.11.0

### PR DESCRIPTION
**Pull request by mlaventure**
Signed-off-by: Kenfe-Mickael Laventure mickael.laventure@gmail.com

---

almost there

### CODE REVIEW

**Comment by tiborvass** [comment]
LGTM

**Comment by tonistiigi** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,137 @@ information on the list of deprecated flags and APIs please have a look at
...
+
+- Fix a bug where Docker would not used the correct uid/gid when processing the `WORKDIR` command ([#21033](https://github.com/docker/docker/pull/21033))
+- Fix a bug where copy operations would not use the proper uid/gid ([#20782](https://github.com/docker/docker/pull/20782), [#21162](https://github.com/docker/docker/pull/21162))
+
+### Client
+
+* Usage of the `:` separator for security option has been deprecated. `=` should be used instead ([#21232](https://github.com/docker/docker/pull/21232))
++ The client user agent is now passed to the registry on `pull`, `build`, `push`, `login` and `search` operations ([#21306](https://github.com/docker/docker/pull/21306), [#21373](https://github.com/docker/docker/pull/21373))
+* Value passed to `--hostname` are now treated as FQDN ([#20200](https://github.com/docker/docker/pull/20200))
+- After a brief memory loss, Docker once more learned to report a container PID statistics ([#21150](https://github.com/docker/docker/pull/21150))
```
Comment (cpuguy83) [inline CHANGELOG.md]: I don't think pid stats ever made it into a release?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,137 @@ information on the list of deprecated flags and APIs please have a look at
...
+* `docker login` now handles token using the implementation found in [docker/distribution](https://github.com/docker/distribution) ([#20832](https://github.com/docker/docker/pull/20832))
+* `docker login` will no longer prompt for an email ([#20565](https://github.com/docker/docker/pull/20565))
+* Docker will now fallback to registry V1 if no basic auth credentials are available ([#20241](https://github.com/docker/docker/pull/20241))
+* Docker will now try to resume layer download where it left timeout after a network error/timeout ([#19840](https://github.com/docker/docker/pull/19840))
+- Fix generated manifest mediaType when pushing cross-repository ([#19509](https://github.com/docker/docker/pull/19509))
+
+### Logging
+
+- Fix a race in the journald log driver ([#21311](https://github.com/docker/docker/pull/21311))
+* The JSON Log driver got a speed boost ([#20605](https://github.com/docker/docker/pull/20605))
```
Comment (cpuguy83) [inline CHANGELOG.md]: This was reverted (because it was slower afterall)

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,137 @@ information on the list of deprecated flags and APIs please have a look at
...
+- Fix a file descriptor leak that would occur every time plugins were enumerated ([#20686](https://github.com/docker/docker/pull/20686))
+- Fix an issue where Authz plugin would corrupt the payload body when faced with a large amount of data ([#20602](https://github.com/docker/docker/pull/20602))
+
+### Runtime
+
++ It is now possible for containers to share the NET and IPC namespaces when `userns` is enabled ([#21383](https://github.com/docker/docker/pull/21383))
++ `docker inspect <image-id>` will now expose the rootfs layers ([#21370](https://github.com/docker/docker/pull/21370))
++ Docker Windows gained a minimal `top` implementation ([#21354](https://github.com/docker/docker/pull/21354))
+* Docker learned to report the faulty exe when a container cannot be started due to its condition ([#21345](https://github.com/docker/docker/pull/21345))
++ `docker -v` now accepts a new flag `nocopy`. This tell the runtime not to copy the container path content into the volume (which is the default behavior) ([#21223](https://github.com/docker/docker/pull/21223))
```
Comment (cpuguy83) [inline CHANGELOG.md]: `docker run -v`

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,137 @@ information on the list of deprecated flags and APIs please have a look at
...
+### Networking
+
+- Fix a bug where IPV6 addresses were not properly handled ([#20842](https://github.com/docker/docker/pull/20842))
+* `docker network inspect` will now report all endpoints whether they have an active container or not ([#21160](https://github.com/docker/docker/pull/21160))
++ Experimental support for the MacVlan and IPVlan network drivers have been added ([#21122](https://github.com/docker/docker/pull/21122))
+* Output of `docker network ls` is now sorted by network name ([#20383](https://github.com/docker/docker/pull/20383))
+- Fix a bug where Docker would allow a network to be created with the reserved `default` name ([#19431](https://github.com/docker/docker/pull/19431))
+* `docker network inspect` now returns whether a network is internal or not ([#19357](https://github.com/docker/docker/pull/19357))
++ Docker learned to create ipv6 enabled networks (`docker network create --ipv6`). This show up as a new  `EnableIPv6` field in `docker inspect`. ([#17513](https://github.com/docker/docker/pull/17513))
+
```
Comment (sanimej) [inline CHANGELOG.md]: - Support for AAAA Records (aka IPv6 Service Discovery) in embedded DNS Server (https://github.com/docker/docker/pull/21396)
- Multiple A/AAAA records from embedded DNS Server for DNS Round robin 
(https://github.com/docker/docker/pull/21019)

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,137 @@ information on the list of deprecated flags and APIs please have a look at
...
 be found.
 
+## 1.11.0 (2016-04-12)
+
+**IMPORTANT**: With Docker 1.11, a docker installation is now made of 4 binaries (`docker`, [`docker-containerd`](https://github.com/docker/containerd), [`docker-containerd-shim`](https://github.com/docker/containerd) and [`docker-runc`](https://github.com/opencontainers/runc)). If you have scripts relying on docker being a single static binaries, please make sure to update them. Interaction with the daemon stay the same otherwise, the usage of the other binaries should be transparent.
+
+### Builder
+
+- Fix a bug where Docker would not used the correct uid/gid when processing the `WORKDIR` command ([#21033](https://github.com/docker/docker/pull/21033))
+- Fix a bug where copy operations would not use the proper uid/gid ([#20782](https://github.com/docker/docker/pull/20782), [#21162](https://github.com/docker/docker/pull/21162))
```
Comment (thaJeztah) [inline CHANGELOG.md]: Should we mention that these issues were when using user-namespaces, or would that be too much information?

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,137 @@ information on the list of deprecated flags and APIs please have a look at
...
+
+### Client
+
+* Usage of the `:` separator for security option has been deprecated. `=` should be used instead ([#21232](https://github.com/docker/docker/pull/21232))
++ The client user agent is now passed to the registry on `pull`, `build`, `push`, `login` and `search` operations ([#21306](https://github.com/docker/docker/pull/21306), [#21373](https://github.com/docker/docker/pull/21373))
```
Comment (thaJeztah) [inline CHANGELOG.md]: Should this be below "Distribution"?

**Pull request closed without merging**